src/cloud_common/protocols/unit/unit_config.py

```diff
@@ -71,6 +71,19 @@ class AlertRulePercents(BaseModel):
     ]
 
 
+class AlertRulePercentsOfDisk(AlertRulePercents):
```

Comment:
suggested renaming AlertRulePartitionUseRatio
"""
Partition use ratio in percents
"""

Reply:
I've merged these changes already into aos_cloud. So is it strong proposal?

Reply:
no. suggestion